mkdocs.yml

```
@@ -1,4 +1,4 @@
-site_name: AI Validation
+site_name: AI Validation Team
 site_description: Documenting the processes of the AI Validation Team at the Ministry of the Interior and Kingdom Relations (Min BZK) in The Netherlands.
```

Just an idea and ok if it's ignored

```suggestion
site_description: Documenting the processes of the AI Validation Team at the Ministry of the Interior and Kingdom Relations in The Netherlands.
```